test/linreg/genData.py

```diff
@@ -0,0 +1,37 @@
+# Samuel Havron
+# https://www.github.com/samuelhavron
+# MIT License
```

Comment:
That license was set by Sam Havron. I have no problem with changing it but we should probably ask Sam first if he is ok with it.